html/modules/custom/ocha_assessments/component/src/ocha-assessments-table.js

```diff
@@ -85,14 +85,14 @@ class OchaAssessmentsTable extends OchaAssessmentsBase {
               r =>
                 html`
                   <tr>
-                    <td><a href="${this.baseurl}/node/${r.nid}">${r.title}</a></td>
-                    <td>${r.field_locations_label}</td>
-                    <td>${r.field_organizations_label}</td>
-                    <td>${r.field_asst_organizations_label}</td>
-                    <td>${r.field_local_groups_label}</td>
-                    <td>${r.field_status}</td>
-                    <td>${this.renderDate(r)}</td>
-                    <td>
+                    <td data-content="Title"><a href="${this.baseurl}/node/${r.nid}">${r.title}</a></td>
+                    <td data-content="Location(s)">${r.field_locations_label}</td>
+                    <td data-content="Managed by">${r.field_organizations_label}</td>
+                    <td data-content="Participating Organization(s)">${r.field_asst_organizations_label}</td>
+                    <td data-content="Clusters/Sectors">${r.field_local_groups_label}</td>
+                    <td data-content="Status">${r.field_status}</td>
+                    <td data-content="Assessment Date(s)">${this.renderDate(r)}</td>
+                    <td data-content="Data">
                       ${this.buildDocument('report', r, 'Report')}
                       ${this.buildDocument('questionnaire', r, 'Questionnaire')}
                       ${this.buildDocument('data', r, 'Data')}
```

Comment:
I suspect this will never be empty, maybe output everything on 1 line like

```
<td>${this.buildDocument('report', r, 'Report')}${this.buildDocument('questionnaire', r, 'Questionnaire')}${this.buildDocument('data', r, 'Data')}</td>
```

Reply:
Ok, thanks for confirming. I'm still not getting data but I don't need to see it right now.